acis_thermal_check/regression_testing.py

```diff
@@ -251,23 +233,26 @@ def compare_results(self, load_week, out_dir):
         new_results = pickle.load(open(new_answer_file, "rb"))
         old_answer_file = os.path.join(test_data_dir, self.name, load_week,
                                        "validation_data.pkl")
-        old_results = pickle.load(open(old_answer_file, "rb"))
+        kwargs = {}
+        if six.PY3:
+            kwargs["encoding"] = "bytes"
+        old_results = pickle.load(open(old_answer_file, "rb"), **kwargs)
         # Compare predictions
         new_pred = new_results["pred"]
-        old_pred = old_results["pred"]
-        pred_keys = set(list(new_pred.keys())+list(old_pred.keys()))
+        old_pred = old_results[b"pred"]
+        pred_keys = set(list(new_pred.keys())+[k.decode() for k in old_pred.keys()])
```

Comment:
This decoding here shouldn't be necessary if using `ascii` encode above (I think!!).  As a side note, I love sets and you can write this more nicely as 
```
pred_keys = set(new_pred) | set(old_pred)
```